src/App.jsx

```diff
@@ -1,13 +1,15 @@
 import { useState } from 'react'
-
 import './App.css'
+import Header from '../components/Header'
```

Comment:
This component that is being imported in doesn't seem to exist on the code for this pr - have you maybe forgotten to add this file to a commit? As-is, the page errors out as it can't find the Header file